presto-docs/src/main/sphinx/presto_cpp/limitations.rst

```diff
@@ -16,6 +16,13 @@ The C++ evaluation engine has a number of limitations:
 
 * Not all built-in types are implemented in C++. Attempting to use unimplemented types will result in a query failure. 
 
+  * All basic and structured types in :doc:`../language/types` are supported, except for ``CHAR``, ``TIME``, and ``TIME WITH TIMEZONE``. 
+    These are subsumed by ``VARCHAR``, ``TIMESTAMP`` and ``TIMESTAMP WITH TIMEZONE``.
+
+  * Presto C++ only supports unlimited length ``VARCHAR``, and does not honor the length ``n`` in ``varchar[n]``.
+
+  * The following types are not supported: ``IPADDRESS``, ``IPPREFIX``, ``UUID``, ``KHYPERLOGLOG``, ``P4HYPERLOGLOG``, ``QDIGEST``, ``TDIGEST``.
```

Comment:
@aditi-pandit it just occurred to me that geometry types are also unsupported, right?

Reply:
Opened #22776 to add.

Reply:
@tdcmeehan : I picked up from https://prestodb.io/docs/0.286/language/types.html. Is there any other reference list ?